ConeSearch.tex

```diff
@@ -1,92 +1,67 @@
-\documentclass[11pt,a4paper]{ivoa}
+\documentclass[11pt,a4paper]{ivoa} 
 \input tthdefs
-
 \usepackage{todonotes}
-\usepackage{enumitem}
+
 \usepackage{listings}
 \lstloadlanguages{XML,sh}
 \lstset{flexiblecolumns=true,tagstyle=\ttfamily, showstringspaces=False}
 
 \usepackage{multirow}
 
-
 \title{Simple Cone Search}
 
 \ivoagroup{Data Access Layer}
 
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/MarcoMolinaro]{Marco Molinaro}
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/AdaNebot]{Ada Nebot}
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/MarkusDemleitner]{Markus Demleitner}
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/BobHanisch]{Robert Hanisch}
 \author[http://www.ivoa.net/twiki/bin/view/IVOA/RayPlante]{Raymond Plante}
+%\author[http://www.ivoa.net/twiki/bin/view/IVOA/MarcoMolinaro]{Marco Molinaro}
+%\author[http://www.ivoa.net/twiki/bin/view/IVOA/MarkusDemleitner]{Markus Demleitner}
+\author[http://www.ivoa.net/twiki/bin/view/IVOA/BobHanisch]{Robert Hanisch}
 \author[http://www.ivoa.net/twiki/bin/view/IVOA/AlexSzalay]{Alex Szalay}
 \author[http://www.ivoa.net/twiki/bin/view/IVOA/RoyWilliams]{Roy Williams}
 
-\editor{Marco Molinaro, Ada Nebot, Raymond Plante}
+\editor{Ray Plante}
```

Comment:
Since you are still editing the document, I think, Marco, that you can add yourself as editor and author. Yes, it is a revert operation, but you still have changes few things not so minor (e.g. errata).

Reply:
OK, done in [f30b86b](https://github.com/ivoa-std/ConeSearch/pull/55/commits/f30b86b58a9a7fd9aa39e3aab2ca99909338e7b7)